components/seoin/Input.tsx

```diff
@@ -1,3 +1,41 @@
-export default function InputSeoin() {
-  return <div>InputSeoin</div>;
+import styles from "./FormSeoin.module.css";
+
+interface InputProps {
+  id: string;
+  type: string;
+  label?: string;
+  value?: string | number;
+  style?: string;
+  isError: boolean;
+  errorText?: string;
+  labelStyle?: string;
+  placeholder: string;
+}
+
+export default function InputSeoin({
+  id,
+  type,
+  label,
+  value,
+  style,
+  isError,
+  errorText,
+  labelStyle,
+  placeholder,
+}: InputProps) {
+  return (
+    <div>
+      <label htmlFor={id} className={labelStyle}>
+        {label}
+      </label>
+      <input
+        id={id}
+        type={type}
+        value={value}
+        className={`${style} ${isError && styles["Error"]}`}
```

Comment:
외부에서 `style prop`으로 커스텀 스타일링 할 수 있어서 좋네요💯
하지만 기본 스타일링은 되어 있으면 더 좋을 거 같아요!
```suggestion
        className={`${styles.input} ${style} ${isError && styles["Error"]}`}
```